flutter/assets/tasks.pbtxt

```diff
@@ -229,12 +229,12 @@ task {
     type: COCOGEN
     full {
       name: "COCO validation set for Stable Diffusion"
-      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_test.tfrecord"
+      input_path: "https://github.com/anhappdev/tmp/releases/download/6/coco_gen_test.tfrecord"
       groundtruth_path: "local:///mlperf_models/stable-diffusion/clip_model_512x512.tflite"
     }
     lite {
       name: "COCO validation set for Stable Diffusion"
-      input_path: "https://github.com/anhappdev/tmp/releases/download/5/coco_gen_full.tfrecord"
+      input_path: "https://github.com/anhappdev/tmp/releases/download/6/coco_gen_full.tfrecord"
       groundtruth_path: ""
     }
     tiny {
```

Comment:
probably we should update the tfrecord file for tiny as well, as the below one will throw an error.

Reply:
Ok. The tiny is for integration test. I will update it later,